integration-tests/test/test_wallets.py

```diff
@@ -86,41 +86,41 @@ def get_rev_address_by_public_key(context: TestingContext, node: Node, public_ke
 def test_alice_pay_bob(started_standalone_bootstrap_node: Node, command_line_options: CommandLineOptions, docker_client: DockerClient, random_generator: Random) -> None:
     with testing_context(command_line_options, random_generator, docker_client) as context:
         transfer_amount = 100
-        alice_rev_address = get_rev_address_by_public_key(context, started_standalone_bootstrap_node, ALICE_KEY.public_key, ALICE_KEY.private_key)
-        bob_rev_address = get_rev_address_by_public_key(context, started_standalone_bootstrap_node, BOB_KEY.public_key, BOB_KEY.private_key)
-        create_genesis_vault(context, started_standalone_bootstrap_node, alice_rev_address, ALICE_GENESIS_VAULT_AMOUNT, ALICE_KEY.private_key)
-        alice_balance = get_vault_balance(context, started_standalone_bootstrap_node, alice_rev_address, ALICE_KEY.private_key)
-        bob_balance = get_vault_balance(context, started_standalone_bootstrap_node, bob_rev_address, BOB_KEY.private_key)
+        alice_rev_address = get_rev_address_by_public_key(context, started_standalone_bootstrap_node, ALICE_KEY.get_public_key().to_hex(), ALICE_KEY.to_hex())
```

Comment:
The `.to_hex()` method should be called right before interpolating the key into a string, not before passing it as a parameter.  Functions like `get_rev_address_by_public_key()` should accept the `PrivateKey` type, not `str`.